### PR TITLE
Add the ability to verify an expected log message was written

### PR DESCRIPTION
**Pull request by Frassle**
We used to test that various errors we're written out to the user in these langhost tests, but at some point those test were removed (I suspect because we stopped printing them to stderr).

This adds in the ability to check for a log message in the engine diagnostics stream, and makes use of that in one of the tests.

### CODE REVIEW

File: sdk/python/lib/test/langhost/util.py [hunk 1]
```diff
@@ -238,10 +247,11 @@ def run_test(self,
             # Tear down the language host process we just spun up.
             langhost.process.kill()
             stdout, stderr = langhost.process.communicate()
+            monitor.server.stop(0)
```
Comment (dixler) [inline sdk/python/lib/test/langhost/util.py:250]: Out of curiosity, why did you move this earlier?

**Review by dixler — APPROVED** [review]
LGTM, but have question around order change.

File: sdk/python/lib/test/langhost/util.py (hunk 1 above)
Comment (Frassle) [reply sdk/python/lib/test/langhost/util.py:250]: We don't need the monitor or engine alive once the langhost has finished, just felt tidy to do all the stopping in one place.

**Comment by Frassle** [comment]
bors merge

**Comment by Frassle** [comment]
windows tests failed

bors retry

**Comment by Frassle** [comment]
bors retry
